src/select/index.tsx

```diff
@@ -34,6 +34,9 @@ const Select = React.forwardRef(
         triggerVariant,
         virtualScroll: restProps.virtualScroll,
       },
+      metadata: {
+        hasInlineLabel: Boolean(restProps.inlineLabelText),
```

Comment:
I think an empty string is as good as not having the text at all, but if we're are looking for if this prop is used or not, this might skew the data.